tsconfig.json

```diff
@@ -20,7 +20,7 @@
     "strict": true,
     "strictNullChecks": true,
     "resolveJsonModule": true,
-    "importsNotUsedAsValues": "error",
+    "verbatimModuleSyntax": true,
```

Comment:
has the same behavior as before but is even more strict (`importNotUsedAsValues`) is now deprecated. This makes us be explicit when importing types:

```
import { ScaleLinear } from "d3-scale"; // error
import { scaleLinear, ScaleLinear } from "d3-scale"; // error

import type { ScaleLinear } from "d3-scale"; // ok
import { scaleLinear, type ScaleLinear } from "d3-scale"; // ok
```

Reply:
Nice